core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -754,7 +746,7 @@ private void catchUp(JournalStateMachine stateMachine, RaftJournalAppender clien
   }
 
   @Override
-  public synchronized void startInternal() throws InterruptedException, IOException {
```

Comment:
I've noticed we have redundant `throws` like this around our codebase too. Do we prefer throwing the most-general exception and require callers to know what to catch? I think we'd prefer having the most-specific exceptions in the method signatures rather than the generic one in order to make it easier to know what exceptions to handle.

Reply:
The function thrown both `InterruptedException` and `IOException`, which is why IntelliJ offered to simplify it.

Reply:
I thought `InterruptedException` was a sub-class of `IOException`; turns out I'm wrong 😓 

`InterruptedException` was never thrown in this method.